README.md

```diff
@@ -89,6 +89,8 @@ A growing set of community-developed and maintained servers demonstrates various
 - **[RAG Web Browser](https://github.com/apify/mcp-server-rag-web-browser)** An MCP server for Apify's RAG Web Browser Actor to perform web searches, scrape URLs, and return content in Markdown.
 - **[XMind](https://github.com/apeyroux/mcp-xmind)** - Read and search through your XMind directory containing XMind files.
 - **[oatpp-mcp](https://github.com/oatpp/oatpp-mcp)** - C++ MCP integration for Oat++. Use [Oat++](https://oatpp.io) to build MCP servers.
+- **[coin_api_mcp](https://github.com/longmans/coin_api_mcp)** - A Model Context Protocol server that provides access to cryptocurrency data. 
```

Comment:
Can you make this description a bit more specific, please (like where the data comes from)? And no need to mention MCP so prominently here—it's assumed 🙂

Reply:
A Model Context Protocol server that provides access to [coinmarketcap](https://coinmarketcap.com/) cryptocurrency data.